Tutorials/Amr/Advection_F/Exec/SingleVortex/inputs.rt

```diff
@@ -1,31 +0,0 @@
-max_step  = 1000000
```

Comment:
Why have these files been removed? `inputs.rt` is needed by our regression tests.